src/Sylius/Behat/Service/Accessor/NotificationAccessor.php

```diff
@@ -91,14 +79,20 @@ public function getMessageType()
             return 'failure';
         }
 
-        return '';
+        throw new \RuntimeException('Cannot resolve message type');
```

Comment:
Why `RuntimeException`? These exceptions should be thrown only when something really bad happens, I would go with some standard exception.